### PR TITLE
HPC-5303 Include quattor patterns

### PR DESCRIPTION
**Pull request by alvarosimon**

### CODE REVIEW

File: tests/logstash_1.2.2.conf [hunk 1]
```diff
@@ -51,6 +51,22 @@ filter {
...
+            "volumeused" => "integer"
+            "volumeavail" => "integer"
+            "volumetotal" => "integer"
+            "objrecovthr" => "integer"
+            "objrecovhz" => "integer"
+            "objdegraded" => "integer"
+            "objtotal" => "integer"
+            "actwrite" => "integer"
+            "actread" => "integer"
+            "actops" => "integer"
```
Comment (itkovian) [inline tests/logstash_1.2.2.conf]: Done in the GROK pattern, so remove.

File: tests/logstash_1.2.2.conf [hunk 2]
```diff
@@ -51,6 +51,22 @@ filter {
...
+    mutate {
+        convert => {
+            "volumedata" => "integer"
+            "volumeused" => "integer"
+            "volumeavail" => "integer"
+            "volumetotal" => "integer"
+            "objrecovthr" => "integer"
+            "objrecovhz" => "integer"
+            "objdegraded" => "integer"
+            "objtotal" => "integer"
```
Comment (itkovian) [inline tests/logstash_1.2.2.conf]: Done in the GROK pattern, so remove.

File: tests/logstash_1.2.2.conf [hunk 3]
```diff
@@ -51,6 +51,22 @@ filter {
...
     }
+
+    mutate {
+        convert => {
+            "volumedata" => "integer"
+            "volumeused" => "integer"
+            "volumeavail" => "integer"
+            "volumetotal" => "integer"
+            "objrecovthr" => "integer"
+            "objrecovhz" => "integer"
```
Comment (itkovian) [inline tests/logstash_1.2.2.conf]: Done in the GROK pattern, so remove.

File: tests/logstash_1.2.2.conf [hunk 4]
```diff
@@ -51,6 +51,22 @@ filter {
...
+            "objdegraded" => "integer"
```
Comment (itkovian) [inline tests/logstash_1.2.2.conf]: Done in the GROK pattern, so remove.

**Comment by alvarosimon** [comment]
@itkovian remarks addressed, I think that's ok now...

**Pull request merged**